pkg/util/etcdutil/migrate.go

```diff
@@ -102,34 +101,22 @@ func waitEtcdTPRReady(restClient restclient.Interface, ns string) error {
 
 func createBootstrapEtcdService(kubecli kubernetes.Interface, svcPath string) error {
 	// Create the service.
-	svc, err := ioutil.ReadFile(svcPath)
+	svcb, err := ioutil.ReadFile(svcPath)
 	if err != nil {
 		return err
 	}
-	if err := kubecli.CoreV1().RESTClient().Post().RequestURI(fmt.Sprintf("/api/v1/namespaces/%s/services", api.NamespaceSystem)).SetHeader("Content-Type", "application/json").Body(svc).Do().Error(); err != nil {
+	if err := kubecli.CoreV1().RESTClient().Post().RequestURI(fmt.Sprintf("/api/v1/namespaces/%s/services", api.NamespaceSystem)).SetHeader("Content-Type", "application/json").Body(svcb).Do().Error(); err != nil {
+		return err
+	}
+
+	svc, err := kubecli.CoreV1().Services(api.NamespaceSystem).Get(bootstrapEtcdServiceName, v1.GetOptions{})
```

Comment:
@diegs 

is there a reason we also put this get func into the wait loop? if 108 returns, we should be able to get the service, right?